Remove xcp-ng-dev-env-create

glehmann opened the pull request:
Now that the images are easily available on ghcr.io.
It makes the completion easier to use for the main tool.

PR stacked on #37

stormi commented:
For a PR that undoes what someone else did, I 'd prefer a discussion first, before a merge request (which, unless stated otherwise, any PR is).

glehmann commented:
I've converted it to a draft.

The discussion started in https://github.com/xcp-ng/xcp-ng-build-env/pull/36#issuecomment-3221270668, and I've created a new PR to ensure the subject isn't hidden in the ghcr.io discussion.

ydirson reviewed (commented):
There are many renames but the reason is not given.  Maybe it is not necessary to revert the whole original commit?

glehmann commented:
> There are many renames but the reason is not given. Maybe it is not necessary to revert the whole original commit?

Ok, I thought it was clear by itself, but I was obviously wrong.

I removed the command to avoid collision with the main tool and because the image could be downloaded from ghcr.io.
I've moved `build.sh` to the root of the repository to make it more accessible than in the subdirectories, where it has no point to be, since there is no (easy) way to call it anymore. And I moved the files linked to the build as well.

This is just acknowledging the new situation with the downloadable image. It indeed results in almost reverting a previous commit. The alternative was to block the PR because this one was in preparation? That doesn't sound any better to me!

psafont commented:
Can these files be moved to a new subdirectory `container`? It makes it easier to associate all these files with the `container` subcommand

glehmann commented:
done
